allensdk/internal/api/mtrain_api.py

```diff
@@ -6,12 +6,14 @@
 import json
 
 from . import PostgresQueryMixin
+from .behavior_api import BehaviorApi
```

Comment:
One thing I didn't see in our earlier review is that this module should be behavior_lims_api, and this class should then be BehaviorLimsApi.  Other than that, looks great.  While you do that refactor, Ill give the code a test-drive

Reply:
Done. One hitch: there was already a test file called 'test_behavior_lims_api.py', but it seemed to be designed to test 'behavior_ophys_api.py'. I changed the existing test file to 'test_behavior_ophys_lims_api.py'

Reply:
Great!